docs/resources/Guild.md

```diff
@@ -1088,6 +1089,40 @@ Returns a PNG image widget for the guild. Requires no permissions or authenticat
 | banner3 | large image with guild icon, name and online count. In the footer, Discord logo on the left and "Chat Now" on the right                                        | [Example](https://discord.com/api/guilds/81384788765712384/widget.png?style=banner3) |
 | banner4 | large Discord logo at the top of the widget. Guild icon, name and online count in the middle portion of the widget and a "JOIN MY SERVER" button at the bottom | [Example](https://discord.com/api/guilds/81384788765712384/widget.png?style=banner4) |
 
+## Get Guild Discovery Metadata % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/discovery-metadata
+
+Returns the [discovery metadata](#DOCS_RESOURCES_DISCOVERY/discovery-metadata-object) object for the guild. Requires the `MANAGE_GUILD` permission.
+
+## Modify Guild Discovery Metadata % PATCH /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/discovery-metadata
+
+Modify the [discovery metadata](#DOCS_RESOURCES_DISCOVERY/discovery-metadata-object) for the guild. Requires the `MANAGE_GUILD` permission. Returns the updated [discovery metadata](#DOCS_RESOURCES_DISCOVERY/discovery-metadata-object) object on success.
+
+> info
+> All parameters to this endpoint are optional and nullable. Omitting or setting a null value will set it to default.
```

Comment:
I'm a bit confused, this is quite inconsistent with the rest of the API. It would be very helpful to not require providing keys that we don't want to change.